torchvision/models/quantization/inception.py

```diff
@@ -22,7 +25,12 @@
 }
 
 
-def inception_v3(pretrained=False, progress=True, quantize=False, **kwargs):
+def inception_v3(
+    pretrained: bool = False,
+    progress: bool = True,
+    quantize: bool = False,
+    **kwargs: Any,
+):
```

Comment:
missing return type: `QuantizableInception3`

Reply:
I cannot add this, since `QuantizableInception3` is declared later in the file.
For this I need to shift this function to end of file. 
Same reason for GooglLeNet, InceptionNet